at-quannguyen/W4/main.rb

```diff
@@ -0,0 +1,59 @@
+require './bbcnews.rb'
+require './cnnnews.rb'
+require './user.rb'
+
+bbc_news_1 = BbcNews.new(1, 'BBC first news', 'Hello world from BBC', 'Say hello')
+bbc_news_2 = BbcNews.new(2, 'BBC second news', 'Hello Ruby from BBC', 'Say hello')
+bbc_news_3 = BbcNews.new(3, 'BBC third news', 'Hello AsianTech from BBC', 'Say hello')
+
+cnn_news_1 = CnnNews.new(1, 'CNN first news', 'Hello world from CNN', '06/07/2018')
+cnn_news_2 = CnnNews.new(2, 'CNN second news', 'Hello Ruby from CNN', '06/07/2018')
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/a7fd7c22501ceb429d470c5982e714f3d16f651e/at-quannguyen/W4/main.rb#L10"></span>Naming/VariableNumber: Use normalcase for variable numbers.</td>
    </tr>
  </tbody>
</table>

Reply:
Chỗ này em thử sửa tên , ở trên đề là cnn_news_01 , em sửa thành cnn_news_1 nhưng vẩn báo lỗi